raphtory-graphql/src/data.rs

```diff
@@ -285,40 +269,43 @@ fn is_disk_graph_dir(path: &Path) -> bool {
     has_disk_graph_files
 }
 
-pub(crate) fn get_graph_name(path: &Path) -> Result<String, &'static str> {
-    path.file_name()
-        .and_then(|os_str| os_str.to_str())
-        .map(|str_slice| str_slice.to_string())
-        .ok_or("No file name found in the path or invalid UTF-8")
+pub(crate) fn get_graph_name(path: &Path) -> String {
+    match path.file_stem() {
+        None => path.display().to_string(), //This will only happen if there is no file name
```

Comment:
What happens for disk graph currently? In that case the path is a directory which may fall in here?

Reply:
Or are they being handled via bincode now?